.circleci/config.yml

```diff
@@ -0,0 +1,66 @@
+# https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs
+defaults: &defaults
+  docker:
+    - image: php:7.1.3-cli
+  working_directory: ~/drush
+  environment:
+    TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    TERM: dumb
+    UNISH_NO_TIMEOUTS: y
+    UNISH_DB_URL: mysql://root:@127.0.0.1
+    UNISH_TMP: /tmp
+    PHPUNIT_ARGS: ""
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    docker:
+      # We can probably use a standard php container here. Maybe we should make our own w/ everything pre-installed
+      - image: circleci/php:7.1-apache-node
```

Comment:
does this override *defaults?

Reply:
Yes. I could remove the image in *defaults, since it is never used.